tools/wptserve/wptserve/config.py

```diff
@@ -61,12 +75,14 @@ def __len__(self):
 
     @property
     def logger(self):
```

Comment:
```suggestion
    def logger(self) -> Something
```

would be helpful here. Or just a comment that says what this thing is. In particular I'm wondering if it's definitely a JSON-serializable value, or what is the thing getting passed between processes?

Reply:
No, this is "a thing that implements the logging API". It's specifically not required to be serializable; the logger is created in the current process (in the case when a name is supplied).

Reply:
OK, so a comment block in a central place explaining how this is all wired together would be helpful. My naive question when arriving at that comment block would be "why doesn't the subprocess just log to stdout and the parent process just copies it line-by-line?"